llumnix/arg_utils.py

```diff
@@ -52,10 +52,11 @@ class EngineManagerArgs:
     profiling_result_file_path: str = ""
 
     gpu_type: str = "a10"
-
-    migration_backend: str = "rpc"
-    migration_cache_blocks: int = 512
-    last_stage_max_blocks: int = 4
+    migration_backend_init_timeout: float = 10.0
+    migration_backend: str = "gloo"
+    migration_cache_blocks: int = 16
```

Comment:
then add a comment below "class EngineManagerArgs" for clarifying these values are default values

Reply:
Dataclass generally provides default values, so there is no need for special explanations.